### PR TITLE
fix(increase-prometheus-storage): remove cronjob

### PR DESCRIPTION
**Pull request by georgettica**
the cronjob was for a one time operation, now that the change is done,
it can be removed

### CODE REVIEW

**Comment by georgettica** [comment]
/hold
didn't delete everything

**Review by NautiluX — APPROVED** [review]
/lgtm
/approve

**Comment by georgettica** [comment]
/unhold
and I need another lgtm @NautiluX

**Pull request merged**
